js/src/block-editor/controls/test/image.js

```diff
@@ -8,6 +8,14 @@ import { render } from '@testing-library/react';
  */
 import GcbImageControl from '../image';
 
+jest.mock( '@wordpress/api-fetch', () => {
```

Comment:
Prevents a Jest warning when this packages calls `window.fetch()`, which isn't defined.